CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# How to Contribute
+
+Interested in contributing to the Airbnb Ruby style guide? Awesome! Here are
+some guidelines to help you out.
+
+**Setting expectations**: Although we love hearing from others who are invested
+in Ruby style, ultimately this style guide is used to dictate the code style
+*within Airbnb*. It isn't a community style guide, and as such we may reject
+issues or PRs that simply don't fit our internal engineering needs.
+
+## Pull Requests
+
+PRs to the style guide should include a description of the proposal, including
+examples of what "good" and "bad" code would look like under the new rule, as
+well as a rationale for the rule in question. The description of the rule and
+the examples belong in `README.md`; the rationale goes in the corresponding
+section in `rationales.md`
```

Comment:
Perhaps add instructions to create a link between the 2 files for each rule or guide created.